### ISSUE TITLE
Import playlist?

### ISSUE THREAD

**Issue by mralusw** [polymorphicshade/Tubular#82]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/Tubular/issues) or [closed](https://github.com/polymorphicshade/Tubular/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for Tubular itself and that requests for adding a new service need to be made in [vanilla NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the vanilla NewPipe [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Can I import playlists from Newpipe, from a list of URLs, from WHATEVER basically? Whenever I share to Tubular it only takes the first Url

### Why do you want this feature?

I have a playlist with 200 URL s in newpipe x SB. And there's no way apparently to get it into Tubular. Newpipe clearly exports a list of URLs, as I can save it to a text file or whatever.

But Tubular
1. has no playlist import from any format (list of URLs 1 per line most logical I think), and 
2. when it receives the list from a share action, only reads the first URL

### Why ist the feature relevant to this fork?

Basically it seems impossible to interoperate with Tubular and get rid of newpipe

**Comment by polymorphicshade** [comment]
Have you tried exporting your NewPipe X SponsorBlock database and importing it in to Tubular?

**Comment by mralusw** [comment]
Yeah. The import / export options are placed in different positions in the menus of the two apps. I couldn't find the Newpipe export back when I installed it...

Now that you mentioned it, I have found the newpipe export. But the databases have diverged in the meantime :(

But why is this happening? Playlists are shared FROM as a list, but shared TO as a single entry (in both apps). Wouldn't it make sense to be consistent? It would also help with importing playlist files.

**Comment by polymorphicshade** [comment]
@mralusw your questions should be directed at the vanilla NewPipe devs (I didn't design how the NewPipe database works).

Nonetheless, I do plan on making importing playlists easier 😁

**Comment by mralusw** [comment]
I have no questions about Newpipe SB. I was describing how I ended up in my situation.

Thank for putting the issue on the planned label.

**Comment by mralusw** [comment]
My question was about the share TO feature as it exists now in Tubular. The sender sends a URL list, bit Tubular ignores everything except the first entry. 

So for example if you shared a playlist from Tubular to Tubular (I know it's not possible directly, but you could via e.g. GKeep or any editor) you would lose every item except the first.
